_pages/en_US/uninstall-cfw.txt

```diff
@@ -6,65 +6,128 @@ title: "Uninstall CFW"
+  + If your console displays the Japanese version of Flipnote Studio, a black screen, or an error message, the test has failed
+1. Power off your device
+
+##### DS Download Play Test
+1. Power on your device
+1. Launch the Download Play application (![]({{ "/images/download-play-icon.png" | absolute_url }}){: height="24px" width="24px"})
+1. Select "Nintendo DS"
+1. If your device loads into a "Download software via DS Download Play" menu, the test was successful
+  + If your console displays the Japanese version of Flipnote Studio, a black screen, or an error message, the test has failed
+1. Power off your device
+
+If either of these tests has failed, DS mode, DS Download Play, and/or DS Connection Settings may be inaccessible once CFW is uninstalled! You should [fix DS mode](troubleshooting#dsi--ds-functionality-is-broken-after-completing-the-guide) before continuing.
+{: .notice--warning}
+
+#### Section III - Safety Test
+The purpose of this section is to verify that the console will boot and that critical system functions, like System Settings and the keyboard, will work once CFW is uninstalled. **If you skip this section, you may BRICK your device!**
+
 1. Launch GodMode9 by holding (Start) during boot
 1. If you are prompted to create an essential files backup, press (A) to do so, then press (A) to continue once it has completed
 1. If you are prompted to fix the RTC date&time, press (A) to do so, then set the date and time, then press (A) to continue
-  + Note that, if you had to fix the RTC date and time, you will have to fix the time in the System Settings as well after this guide
-1. Navigate to `[1:] SYSNAND CTRNAND > title > 00040138 > (X)0000002 > content`
-  + On a "New" edition console, the (X) will instead be 2
-  + On an "Old" edition console, the (X) will instead be 0
-1. Select the `.app` file
-1. Select `NCCH image options`
-1. Select `Mount image to drive`
-1. In the window that appears, open the `exefs` folder
-1. In this folder, select the `.firm` file
-1. Select "FIRM image options"
-1. Select "Boot FIRM"
-1. When prompted, choose to decrypt and make a copy
-1. You should boot into the regular 3DS Home Menu. If you do, power your device off and continue to the next section
-  + If you do NOT boot into the regular 3DS Home Menu, continuing with these instructions **WILL** brick your device. You should join [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp) and ask (in English) for someone there to assist you
-  
-#### Section II - Prep Work
+  + Note that, if you had to fix the RTC date and time, you will have to fix the time in the System Settings as well after following this guide
+1. Press (Home) to bring up the action menu
+1. Select "Scripts..."
+1. Select "safety_test"
+1. Read the text on-screen and press (A) to continue
+1. You should boot into the regular 3DS Home Menu. If you do, continue these instructions
+  + If you reach an error screen, uninstalling CFW **WILL BRICK YOUR DEVICE!** 
```

Comment:
This should probably mention the possibility of a black screen as well, possibly "If you reach an error screen, or your device does not turn on..."?